lib/components/admin/mailables-window.js

```diff
@@ -43,64 +45,76 @@ class MailablesWindow extends Component {
 
   _onClickCreateLetter = () => createLetter(this.state, this.props.mailablesConfig)
 
-  _updateField = (evt) => {
+  _updateLetterField = (evt) => {
     this.setState({[evt.target.id]: evt.target.value})
   }
 
   render () {
-    const {mailablesConfig, callTaker, toggleMailables} = this.props
+    const {mailablesConfig, mailablesEnabled, callTaker, toggleMailables} = this.props
+    if (!mailablesEnabled) return null
     const {mailables: selectedMailables} = this.state
     const {items} = mailablesConfig
     if (!callTaker.mailables.visible) return null
     const selectableMailables = items.filter(m =>
       !selectedMailables.find(mailable => mailable.name === m.name)
     )
+    const MAILABLE_WIDTH = 300
     return (
       <DraggableWindow
+        footer={
+          <>
+            <span className='pull-right'>
```

Comment:
The `<>` tag is not needed. The extra `<span>` tag is possibly not needed (can the `<Button>` be of class `pull-right`?).